### DISCUSSION TITLE
Why does ohmyposh need internet access?

### DISCUSSION THREAD

**Discussion by rosmur** [JanDeDobbeleer/oh-my-posh#5653] [General]
I get the following popup when using ohmyposh

![image](https://github.com/user-attachments/assets/7bc48d30-86ea-430f-941f-e6637e827465)

Why does ohmyposh need internet access? P.S: I changed my shell configuration to use a local copy of the theme i.e. local file as arg for --config.

**Comment by JanDeDobbeleer** [comment]
@rosmur it checks if it needs to display an upgrade message. Will be disabled by default in the next major version.

**Comment by rosmur** [comment]
Great, thanks! It is very disconcerting to see the command line prompt needing internet access and is a security risk
